kaos/models/models.py

```diff
@@ -33,7 +33,7 @@ class Satellite(SavableModel, DB.Model):
     platform_name = DB.Column(DB.String(50), nullable=False)
     orbit_segments = DB.relationship("OrbitSegment", backref='satellite', lazy=True)
     orbit_records = DB.relationship("OrbitRecord", backref='satellite', lazy=True)
-    maximum_altitude = DB.Column(DB.Float, nullable=True)
+    maximum_altitude = DB.Column(DB.Float)
```

Comment:
I'm a little confused by this. In models/parser.py:67-69, we create the satellite as follows:

```
    sat = Satellite(platform_name=os.path.splitext(os.path.basename(filename))[0])
    sat.save()
    DB.session.commit()
```

The `maximum_altitude` is not specified until later. Shouldn't making it non-nullable cause an error with our current parser?

Reply:
Nope, when you create a value it still gets a default value ( I believe 0 )